pkg/controllers/job/job_controller_handler.go

```diff
@@ -241,7 +247,9 @@ func (cc *Controller) updatePod(oldObj, newObj interface{}) {
 		JobVersion: int32(dVersion),
 	}
 
-	cc.queue.Add(req)
+	key := vkcache.JobKeyByReq(&req)
+	i := cc.getWorkerID(key)
```

Comment:
I think this can guarantee worker threads will not process same jobs

Reply:
yes